prisma/migrations/20240613093958_initial_migrations/migration.sql

```diff
@@ -0,0 +1,79 @@
+-- CreateTable
+CREATE TABLE "birthday" (
+    "user_id" VARCHAR(19) NOT NULL,
+    "guild_id" VARCHAR(19) NOT NULL,
+    "birthday" VARCHAR(10) NOT NULL,
+    "disabled" BOOLEAN NOT NULL DEFAULT false,
+    "created_at" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" TIMESTAMP(3) NOT NULL
+);
+
+-- CreateTable
+CREATE TABLE "guild" (
+    "id" VARCHAR(19) NOT NULL,
+    "inviter" VARCHAR(19),
+    "announcementChannel" VARCHAR(19),
+    "announcementMessage" VARCHAR(512),
+    "overviewChannel" VARCHAR(19),
+    "overviewMessage" VARCHAR(19),
+    "birthday_role" VARCHAR(19),
+    "birthday_ping_role" VARCHAR(19),
+    "log_channel" VARCHAR(19),
+    "timezone" INTEGER NOT NULL DEFAULT 0,
```

Comment:
Do we / Should we use timezone offsets or IANA Timezone Names in the DB?

Reply:
I would say directly the IANA Timezone Names would be easier to manage I think